### ISSUE TITLE
Audio is very high in general compared to everything else

### ISSUE THREAD

**Issue by Gagert** [dethrace-labs/dethrace#342]
As the title says, I have to turn my audio down to a barely noticeable volume in Windows 11 when playing both the latest released build (0.6.0) & in the latest compiled build (4200cfa) of Dethrace because of the audio volume differences. If I don't touch the volume slider in Windows 11 before starting this, the audio in Dethrace becomes very high.

I tried adjusting the sound volume in the options.txt file inside the data folder (evolume & mvolume), but it didn't seem to make any difference.

**Comment by jeff-1amstudios** [comment]
Yes, I have this too on my system (macOS). There is a "--volume-multiplier=" argument you can pass to change it. Try values between 0.1 and 1. 

What multiplier sounds “about right” to you?

**Comment by Gagert** [comment]
I think that this might as well have been a false positive on my behalf; the menu clicking-sounds are very loud compared to the in-game audio, which in fact is okay in my opinion. I may have assumed that the in-game audio would blow my speakers judging from the menu sound-effects, but I in fact find the in-game audio less loud.

After hearing the loud menu noises, as a reaction I turned the volume on my receiver way down just to be safe before I started a race.

Feel free to close this Issue, my bad.
